Big-Spender/readme.md

```diff
@@ -124,7 +124,8 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+insert into spends (date, transaction_no, supplier_inv_no, description, amount)
+values ('2021-08-19', '38104091', '3780119655', 'Computer Hardware Dell' 32000);
```

Comment:
After the session, do you think you have a better idea of how we could do this insert into statement? any ideas what might be the best way to do this?